.github/workflows/build.yaml

```diff
@@ -25,6 +25,4 @@ jobs:
       - name: Build the container
         run: |
           make build-container
-      - name: Run the container
-        run: |
-          docker run quay.io/operator-framework/combo:dev
+
```

Comment:
I had mentioned this in https://github.com/operator-framework/combo/pull/24/files#r747829764 but it looks like I was reviewing a potential out-of-date diff at that time - I'd like to keep this check to avoid regressing on the overall container image being unable to run successfully.

Reply:
Brought this back now!